frontend/src/features/map/layers/styles/isForbiddenPeriod.test.ts

```diff
@@ -551,6 +551,44 @@ describe('isForbiddenPeriod', () => {
     const currentDate = getUtcDayjs().day(1) // Monday
     expect(isForbiddenPeriod(feature, currentDate)).toEqual(true)
   })
+
+  it('isForbiddenPeriod Should return false When authorized is false and dates contains a null item', async () => {
```

Comment:
In Jest (but also in Mocha, AVA, etc) we normally don't repeat the name of the function in the `it` since it should already be declared in the `describe`:

```
describe('myFunction()', () => {
  it('should do that',  () => ...)
  it('should also do that',  () => ...)
 })
```

Reply:
I'm used to having describe with multiple functions to test, I removed the repeat of the function name